### PR TITLE
Add new `entries` command to `content-generator`

### PR DESCRIPTION
**Pull request by dimas-b**
Useful tool for exploring repositories when java runtime is available, but not python.

### CODE REVIEW

File: tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+package org.projectnessie.tools.contentgenerator.cli;
+
+import java.util.List;
+import org.projectnessie.client.api.NessieApiV2;
+import org.projectnessie.error.NessieNotFoundException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(
+    name = "list-entries",
```
Comment (adutra) [inline tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java:25]: Not a big deal, but other similar reading commands do not have the prefix `list-` or `read-`. How about renaming to just `entries`?

File: tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  @Option(
+      names = {"-C", "--with-content"},
+      description = "Include content for each entry.")
+  private boolean withContent;
+
+  @Override
+  public void execute() throws NessieNotFoundException {
+    try (NessieApiV2 api = createNessieApiInstance()) {
+      spec.commandLine().getOut().printf("Listing entries for reference '%s'.%n%n", ref);
```
Comment (snazy) [inline tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java:43]: Nit: extract `spec.commandLine().getOut()` to a variable.

**Review by snazy — DISMISSED** [review]

File: tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java (hunk 1 above)
Comment (dimas-b) [reply tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java]: renamed

File: tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java (hunk 2 above)
Comment (dimas-b) [reply tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/ReadEntries.java]: done

**Review by adutra — APPROVED** [review]

**Pull request merged**